src/app/test-results/test-results.module.ts

```diff
@@ -0,0 +1,20 @@
+import { AppUtilsService } from './../app-utils.service';
+
+import { TableModule } from 'primeng/table';
```

Comment:
nit: Excess spacing. Suggest removing to stay consistent with the other ts files

```suggestion
import { AppUtilsService } from './../app-utils.service';
import { TableModule } from 'primeng/table';
```